src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.Builder.cs

```diff
@@ -239,6 +239,61 @@ public void Insert(int index, T item)
                 _elements[index] = item;
             }
 
+            /// <summary>
+            /// Inserts the specified values at the specified index.
+            /// </summary>
+            /// <param name="index">The index at which to insert the value.</param>
+            /// <param name="items">The elements to insert.</param>
+            public void InsertRange(int index, IEnumerable<T> items)
+            {
+                Requires.Range(index >= 0 && index <= this.Count, nameof(index));
+                Requires.NotNull(items, nameof(items));
+
+                int count = ImmutableExtensions.GetCount(ref items);
+                this.EnsureCapacity(this.Count + count);
```

Comment:
Actually, please disregard. It seems that this is convention for the given project, so it might make sense to preserve consistency with the rest of the implementation file.